HeterogeneousCore/AlpakaCore/interface/alpaka/Event.h

```diff
@@ -47,6 +47,10 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::device {
     auto streamID() const { return constEvent_.streamID(); }
     auto id() const { return constEvent_.id(); }
 
+    // To be able to interact with non-Alpaka helper code that needs
+    // to access edm::Event
+    edm::Event const& edmEvent() const { return constEvent_; }
```

Comment:
I'm fine with an explicit method like this.

So, just checking, what do you think about a conversion specifier like
```c++
operator edm::Event const&() const { return constEvent_; }
```
?

This would allow passing a `device::Event` transparently where an `edm::Event const&` is expected.

Reply:
I thought about this question and went back and forth, and in the end settled in favoring user convenience (which has been a theme for this version of the design), given that I could not think of a way how the implicit conversion could be misused or really caused unexpected behavior.